### PR TITLE
Add default dashboard template 101-default-shared-dashboard as a comm…

### PR DESCRIPTION
**Pull request by mbenko**
…on starting point for creating a sharable view of an application from the Azure portal.

### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog
* Added a new template for deploying a shared dashboard that provides a view into a given resource group.

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (bmoore-msft) [inline .vs/VSWorkspaceState.json]: remove all of the files in the .vs folder

File: 101-default-shared-dashboard/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
+{
```
Comment (bmoore-msft) [inline 101-default-shared-dashboard/azuredeploy.json]: format the json, e.g. SHIFT+ALT+F in VS Code

File: 101-default-shared-dashboard/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```
Comment (bmoore-msft) [inline 101-default-shared-dashboard/azuredeploy.json]: Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

File: 101-default-shared-dashboard/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    "dashboardName": { "type": "string" }
+  },
+  "variables": {
+  },
+  "resources": [
+    {
+      "properties": {
```
Comment (bmoore-msft) [inline 101-default-shared-dashboard/azuredeploy.json]: move the properties node to the bottom of the resource

File: 101-default-shared-dashboard/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dashboardName": { "type": "string" }
```
Comment (bmoore-msft) [inline 101-default-shared-dashboard/azuredeploy.json]: add a defaultValue

File: 101-default-shared-dashboard/azuredeploy.parameters.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dashboardName": {"value":"Shared Dashboard"}
```
Comment (bmoore-msft) [inline 101-default-shared-dashboard/azuredeploy.parameters.json]: use GEN-UNIQUE

File: 101-default-shared-dashboard/azuredeploy.json (hunk 3 above)
Comment (mbenko) [reply 101-default-shared-dashboard/azuredeploy.json]: The dashboard resource is deployed into the selected resource group, shouldn't it stay in that location?

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (mbenko) [reply .vs/VSWorkspaceState.json]: done.

File: 101-default-shared-dashboard/azuredeploy.json (hunk 2 above)
Comment (mbenko) [reply 101-default-shared-dashboard/azuredeploy.json]: done

File: 101-default-shared-dashboard/azuredeploy.json (hunk 4 above)
Comment (mbenko) [reply 101-default-shared-dashboard/azuredeploy.json]: done

File: 101-default-shared-dashboard/azuredeploy.json (hunk 5 above)
Comment (mbenko) [reply 101-default-shared-dashboard/azuredeploy.json]: done

File: 101-default-shared-dashboard/azuredeploy.parameters.json (hunk 6 above)
Comment (mbenko) [reply 101-default-shared-dashboard/azuredeploy.parameters.json]: done

**Comment by mbenko** [comment]
I updated the template with the changes from above except for the one on Location. Since the dashboard is deployed to the selected Resource Group (it is a default dashboard for that specific RG) shouldn't it be deployed into the same location? Why would they want to deploy it to somewhere else?

**Pull request merged**